### PR TITLE
GOLDS-352 - fix updating Leafspace satellite/groundstation links + refactor names of Leafspace resources

### PR DESCRIPTION
**Pull request by leanspace-thomas**

### CODE REVIEW

File: services/importer.go [hunk 1]
```diff
@@ -60,8 +60,8 @@ func AddDataTypes() {
 	routes.RouteDataType.Subscribe()
 	processors.ProcessorDataType.Subscribe()
 	orbit_resources.OrbitResourceDataType.Subscribe()
-	leaf_space_connections.LeafSpaceIntegrationDataType.Subscribe()
-	leafSpaceGroundstation.LeafSpaceGroundStationLink.Subscribe()
-	leafSpaceSatellite.LeafSpaceSatellitesLink.Subscribe()
-	contact_reservations.LeafSpaceContactReservation.Subscribe()
+	leaf_space_connections.LeafSpaceConnectionDataType.Subscribe()
+	leafSpaceGroundstationLink.LeafSpaceGroundStationLinkDataType.Subscribe()
+	leafSpaceSatelliteLink.LeafSpaceSatellitesLinkDataType.Subscribe()
```
Comment (jerome-leanspace) [inline services/importer.go:65]: if you're already changing them, can you use snake case?

File: services/leaf_space_integration/link/groundstation/models.go [hunk 2]
```diff
@@ -1,13 +1,13 @@
-package leafSpaceGroundstation
+package leafSpaceGroundstationLink
```
Comment (jerome-leanspace) [inline services/leaf_space_integration/link/groundstation/models.go:1]: pakages should also be snake case

**Review by jerome-leanspace — APPROVED** [review]
the actual change looks good
The semantic changes needs to be snake case most of the time

File: services/importer.go (hunk 1 above)
Comment (leanspace-thomas) [reply services/importer.go]: Ok done.

File: services/leaf_space_integration/link/groundstation/models.go (hunk 2 above)
Comment (leanspace-thomas) [reply services/leaf_space_integration/link/groundstation/models.go]: Ok done.

**Review by jerome-leanspace — APPROVED** [review]

**Pull request merged**